pkgs/tools/package-management/nix/common.nix

```diff
@@ -15,14 +15,8 @@ let
   atLeast210 = lib.versionAtLeast version "2.10pre";
   atLeast213 = lib.versionAtLeast version "2.13pre";
   atLeast214 = lib.versionAtLeast version "2.14pre";
+  atLeast220 = lib.versionAtLeast version "2.20pre";
   atLeast221 = lib.versionAtLeast version "2.21pre";
-  # Major.minor versions unaffected by CVE-2024-27297
-  unaffectedByFodSandboxEscape = [
-    "2.3"
-    "2.18"
-    "2.19"
-    "2.20"
-  ];
```

Comment:
Should we remove this in the original PR, too? I am not sure.

Reply:
No, because then you need to upgrade **ALL** the other Nix. Not just one.

Reply:
Good, then we leave it as it is in my PR. I just wasn't sure if I missed something or this was a bad rebase.